CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 All notable changes to the Zowe API Mediation Layer package will be documented in this file.
 
+## `1.24.4 (2021-09-01)`
+
+* Feature:  SAF resource access check controller (#1734) ([680f4b9](https://github.com/zowe/api-layer/commit/680f4b9)), closes [#1734](https://github.com/zowe/api-layer/issues/1734)
```

Comment:
Can you add a little detail about the benefit of this feature so we can add this to the Summary of Changes?

Reply:
I changed it a bit. It's undocumented at the moment because it's been done for a specific frontend case. When properly released, it might change a bit and full documentation will be available.